src/tensor_ops/reshape_to/cuda_kernel.rs

```diff
@@ -102,3 +83,97 @@ where
         Ok(())
     }
 }
+
+const FWD_KERNEL: &str = "
+#if __WORDSIZE == 64
+typedef long int intptr_t;
+#else
+typedef int intptr_t;
+#endif
```

Comment:
Noting that you can't use `<cstdin>` or `"stdint.h"` in nvrtc because of include paths.

According to https://forums.developer.nvidia.com/t/nvrtc-missing-stdint-h/146318, including standard headers like this should be avoided. The only reason we need the above is for intptr_t, so its easy to redefine them as the above.